### DISCUSSION TITLE
CKauth

### DISCUSSION THREAD

**Discussion by zumwaltboi** [3kh0/ext-remover#666] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My school has a proxy extension called Ckauththenticator G3 (contentkeeperG3) none of the exploits work to bypass it. I can turn it off with the killcurly exploit, but websites are still restricted by my schools network proxy. And at home the websites wont connect. Any fixes?

**Comment by FearedFusionX** [comment, marked as answer]
Try using the LTMEAT exploit and kill the manifest of that extension. If that doesn't work, Idk.
